matrix.js

```diff
@@ -153,7 +165,10 @@ define([], function(){
 			} else {
 				var pivot = _pivot(U, i);
 				if(pivot == i)
-					throw new Error("Singular matrix, LU decomposition not feasible");
+					throw {
+						type: "decomposition",
+						message: messages.get("decomposition");
```

Comment:
semicolon

Reply:
good catch. I fixed it in pull request number 5. Sorry it was left here.